custom_code/mmt.py

```diff
@@ -225,33 +220,31 @@ def layout(self):
     def observation_payload(self):
         target = Target.objects.get(pk=self.cleaned_data['target_id'])
         ra, dec = SkyCoord(target.ra, target.dec, unit='deg').to_string('hmsdms', sep=':', precision=1).split()
-        maskid = {
-            'Longslit0_75': 113,
-            'Longslit1': 111,
-            'Longslit1_25': 131,
-            'Longslit1_5': 114,
-            'Longslit5': 121,
-        }.get(self.cleaned_data['slit_width'])
+        maskid = 111
+        
```

Comment:
You can just delete this line because it's hardcoded below.

Reply:
Done